eventOrganizer/src/main/java/com/event/organizer/api/appuser/AppUserService.java

```diff
@@ -22,6 +22,17 @@ public class AppUserService implements UserDetailsService {
     private final ConfirmationTokenService confirmationTokenService;
 
     private final PasswordEncoder passwordEncoder;
+
+    public void EditAccount(AppUser editedUser) throws UsernameNotFoundException {
+        if(editedUser == null){
+            throw new UsernameNotFoundException("User is not found.");
+        }

```

Comment:
also add check if edited user is the User who is currently logged in
if(editedUser != loggedInUser) {
     throw new Exception("Permission denied.") 
} 

This is currently blocked by the EV-3 story.